src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ERR.cs

```diff
@@ -21,14 +21,16 @@ internal static partial class Crypto
         internal static extern IntPtr ErrReasonErrorString(ulong error);
 
         [DllImport(Libraries.CryptoNative, CharSet = CharSet.Ansi)]
-        private static extern void ErrErrorStringN(ulong e, [Out] StringBuilder buf, int len);
+        private static unsafe extern void ErrErrorStringN(ulong e, byte* buf, int len);
```

Comment:
> Same question above in ObjObj2Txt

That one's a little different.  For the 99% case (< 80 bytes), we can just allocate it on the stack, which means we have a ```byte*```.  The choice then is whether to have a second P/Invoke signature that takes a ```byte[]``` to handle the fallback case, and that doesn't seem worthwhile when we already have one we can use with ```fixed```.